src/redux/initialState.js

```diff
@@ -34,8 +44,13 @@ const initialState = {
       price: 30,
       stars: 2,
       promo: 'sale',
+      userRating: 3,
       newFurniture: true,
       isFavorite: false,
+      isFeatured: true,
```

Comment:
Może dla ścięcia ilości kodu i trochę logiki warto by było pomyśleć o przeniesieniu tych 4 stanów do jednego w postaci tablicy? Bo jak by była jakaś właściwość produktu zawierająca tablicę tylko z tymi kategoriami które dotyczą danego produktu to powinno się łatwiej tym manipulować np. gdy jeden mebel by miał pustą tablicę to nie musiał by mieć 4 false. Zwłaszcza, że już nawet zrobiłeś tablicę tags i będąc na danej zakładce zaciągałbyś wszystkie dane ze store, szedł po tablicy w każdym produkcie i sprawdzał czy istnieje tam taki element.

Warto by też zrobić, że jak zmienia się kategorię to zmienia się też automatycznie produkt na pierwszy pasujący z danej kategorii.

Reply:
Btw, jak już byś chciał to grupować to jako obiekt a nie tablica. Dostaniesz tablice `[false, true]` a w innym produkcie `[false, false, true]` i niby co to oznacza, za co odpowiada pierwsze false a za co drugie? Nie wiadomo. 

W tagach może tak być, bo dla developera tagi nie są ważne, ktoś tam w tagach może sobie podać co mu się podoba, my po prostu zrobimy po nich pętlę i je wyświetlimy.

Więc to co jest tagami (feature, newFurniture) możemy przenieść do tablicy, ale isFavourite, isCompared itp zostaje tak jak jest moim zdaniem.

Reply:
Aaa ok to jak najbardziej